src/config/__tests__/index.test.ts

```diff
@@ -9,8 +9,13 @@ describe('Config getters', () => {
     })
 
     it('gets a public RPC from the config', () => {
-      setChainId(CHAIN_ID.POLYGON)
-      expect(getPublicRpcUrl()).toBe('https://polygon-rpc.com/')
+      setChainId(CHAIN_ID.ETHEREUM)
+      expect(getPublicRpcUrl()).toMatch('https://mainnet.infura.io/v3/')
+    })
+
+    it('returns nothing for a non-existent config', () => {
```

Comment:
I don't think we even need this test anymore. We can then remove `UNKNOWN` too.

Reply:
UNKNOWN if used in a few other places. Why do you think we don't need this test? I wrote it quite recently, and removing it will decrease our coverage.

Reply:
We don't use local configs anymore. If unknown is used elsewhere though, let's leave it.

Reply:
This test is for the remote config.